Makefile

```diff
@@ -14,8 +14,9 @@ install: install-validator install-node install-python install-hooks
 
 build: build-specification build-coordinator build-validator build-proxies
 
-test: lint validate-models check-licenses test-coordinator
```

Comment:
Removal of lint here as lint calls build and the ADO pipeline calls lint then test so the pipeline is building everything twice. Todo: check commit hook still lints

Reply:
checked, hook calls lint directly